ipalib/install/certmonger.py

```diff
@@ -309,7 +310,8 @@ def request_and_wait_for_cert(
     reqId = request_cert(nssdb, nickname, subject, principal,
                          passwd_fname, dns, ca, profile,
                          pre_command, post_command)
-    state = wait_for_request(reqId, timeout=60)
+    timeout = api.env.startup_timeout + 60
```

Comment:
I still see hardcoded value (60)

AFAIK startup_timeout is 5 minutes yb default, so there is no need for additional 60 seconds